flytekit/core/dynamic_workflow_task.py

```diff
@@ -1,3 +1,23 @@
+"""
+Dynamic Workflows
+-----------------------------
+Dynamic workflows are one of the powerful aspects of Flyte. Please take a look at the :py:func:`flytekit.dynamic` documentation first to get started.
+
+
+Declaring a dynamic workflow
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: flytekit.core.dynamic_workflow_task.DynamicWorkflowTask
+
+
+Caveats when using a dynamic workflow
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Because of the dynamic nature of the workflow generated, it can easily be abused. Keep in mind that the workflow
+that's compiled out of the decorated function needs to be processed like any other workflow. It's rare to see a
+manually written workflow that has 5000 nodes for instance, but you can easily get there with a loop. Please keep,
```

Comment:
should we add a TODO here to update w/ map tasks when available if you are doing some massively parallel thing

Reply:
but this will be done in like three days right?  we'll remember.